src/dash/DashAdapter.js

```diff
@@ -465,6 +467,7 @@ function DashAdapter() {
             event.calculatedPresentationTime = calculatedPresentationTime;
             event.messageData = messageData;
             event.presentationTimeDelta = presentationTimeDelta;
+            event.value = (schemeIdUri === 'https://aomedia.org/emsg/ID3') ? getId3Frames(messageData) : null;;
```

Comment:
Two minor comments:

* There are two `;;` in the end
* Can we move `https://aomedia.org/emsg/ID3` to `Constants.js`